client/bindings/nodejs/types/outputBuilderOptions/index.ts

```diff
@@ -3,7 +3,13 @@ import type { IAliasOutputBuilderOptions } from './aliasOutputOptions';
 import type { IFoundryOutputBuilderOptions } from './foundryOutputOptions';
 import type { INftOutputBuilderOptions } from './nftOutputOptions';
 
+/*
+ * Hex encoded bytes.
+ */
+declare type HexEncodedString = string;
```

Comment:
Why not the one from iota.js then ?

Reply:
Because it's not in the types and we would need to import the whole iota.js lib just for this single type

Reply:
Maybe it's worth asking them to move it then

Reply:
https://github.com/iotaledger/iota.js/issues/1164